common/supplemental/likelySubtags.xml

```diff
@@ -2139,7 +2139,7 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="saf" to="saf_Latn_GH"/>
 		<!--{ Safaliba; ?; ? } => { Safaliba; Latin; Ghana }-->
 		<likelySubtag from="sah" to="sah_Cyrl_RU"/>
-		<!--{ Sakha; ?; ? } => { Sakha; Cyrillic; Russia }-->
+		<!--{ Yakut; ?; ? } => { Yakut; Cyrillic; Russia }-->
```

Comment:
At first I thought there was a problem with alphabetical order here; now I see the ordering is by code, and "Yakut" has the code "saq"; "Yakut, also known as Yakutian, Sakha, Saqa or Saxa" per https://en.wikipedia.org/wiki/Yakut_language so OK!